core/src/main/scala/org/apache/spark/ui/jobs/UIData.scala

```diff
@@ -97,6 +97,7 @@ private[spark] object UIData {
     var memoryBytesSpilled: Long = _
     var diskBytesSpilled: Long = _
     var isBlacklisted: Int = _
+    var jobLastUpdateTime: Option[Long] = None
```

Comment:
Is it better to rename to `stageLastUpdateTime` or just `lastUpdateTime`? Since this structure unrelated to job, would be better to not involve "job".